scripts/auto-update-ssl-cert.sh

```diff
@@ -68,16 +69,17 @@ seconds_before_expire=$(( $DAYS_BEFORE_EXPIRE * 24 * 60 * 60 ));
 $( [[ -e /usr/bin/python ]] || ln -s /usr/bin/python3 /usr/bin/python )
 [[ -f "/var/lib/jelastic/SSL/jelastic.crt" && "$withExtIp" != "false" ]] && exp_date=$(jem ssl checkdomain | python -c "import sys, json; print (json.load(sys.stdin)['expiredate'])");
 
-[ -z "$exp_date" ] && { echo "$(date) - no certificates for update" >> /var/log/letsencrypt.log; exit 0; };
+[ -z "$exp_date" ] && { echo "$(date) - no certificates for update" >> ${LOG_FILE}; exit 0; };
 
 _exp_date_unixtime=$(date --date="$exp_date" "+%s");
 _cur_date_unixtime=$(date "+%s");
 _delta_time=$(( $_exp_date_unixtime - $_cur_date_unixtime  ));
 
 [[ $_delta_time -le $seconds_before_expire ]] && {
-    echo "$(date) - update required" >> /var/log/letsencrypt.log;
+    echo "$(date) - update required" >> ${LOG_FILE};
```

Comment:
```suggestion
    echo "$(date) - update required";
```
+ change in all other places